src/seml/database.py

```diff
@@ -18,6 +20,44 @@ def get_collection(collection_name, mongodb_config=None, suffix=None):
     return db[collection_name]
 
 
+def export_collection(collection_name, path=None):
+    collection = get_collection(collection_name)
+    cursor = collection.find({})
+    if path == None:
+        path = f'{collection_name}.bson'
+    with open(path, 'w') as file:
+        file.write(dumps(cursor))
+    logging.info(
+        f"Successfully exported {collection_name} to {path}.")
+
+
+def import_collection(collection_name, path):
+    mongodb_config = get_mongodb_config()
+    db = get_database(**mongodb_config)
+
+    if collection_name in db.list_collection_names():
+        logging.error(f"Collection {collection_name} already exists.")
+        return
```

Comment:
why shouldn't we support adding to collections? We have to resolve id conflicts?

Reply:
see discussion below